compose/project.py

```diff
@@ -215,6 +221,14 @@ def build(self, service_names=None, no_cache=False, pull=False, force_rm=False):
             else:
                 log.info('%s uses an image, skipping' % service.name)
 
+    def create(self, service_names=None, strategy=ConvergenceStrategy.changed, do_build=True):
+        services = self.get_services_without_duplicate(service_names, include_deps=True)
+
+        plans = self._get_convergence_plans(services, strategy)
+
+        for service in services:
+            service.execute_convergence_plan(plans[service.name], do_build, start=False)
```

Comment:
Just one more small thing, I think we want to avoid calling attach here (it's an unnecessary API call), so this should add `detached=True`  as well